apps/ca.c

```diff
@@ -1709,7 +1709,15 @@ static int do_body(X509 **xret, EVP_PKEY *pkey, X509 *x509,
 
     /* Initialize the context structure */
     X509V3_set_ctx(&ext_ctx, selfsign ? ret : x509,
```

Comment:
Perhaps add a comment that all the necessary information is in `ret`, so there's no need to also provide `req` (assuming this is why `req` was replaced with NULL in the arg list).

Reply:
Good point - done.